askup/utils/views.py

```diff
@@ -305,22 +341,25 @@ def apply_filter_to_queryset(request, filter, queryset):
     return queryset
 
 
-def do_user_checks_and_evaluate(user, answer, evaluation):
-    """Do user checks and evaluate answer for the answer evaluation view."""
+def do_user_checks_and_evaluate(user, answer, evaluation, qset_id):
+    """
+    Do user checks and evaluate answer for the answer evaluation view.
+    """
     evaluation_int = int(evaluation)
     is_admin = check_user_has_groups(user, 'admin')
+    user_permitted = Organization.objects.filter(qset__in=[qset_id], users__in=[user.id]).first()
```

Comment:
For checking, if an object is in the database `exists` works better than `first`.